PatientRegistrationApp/src/components/SinglePatientPersonalInfo.vue

```diff
@@ -218,6 +219,19 @@ export default {
         valid = false;
       }
 
+      if( this.patientPhoto ) {
+        if (this.patientPhoto.size > 2097152) {
+          if (!valid) {
+            message += "\n";
+            message += "Photo is over 2mb's";
+          }
+          else {
+            message = "Photo is over 2mb's";
```

Comment:
Can we change the message to read "Your selected photo is too large. Please resubmit one under 2MBs."